CotEditor/en-GB.lproj/Acknowledgments.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8"/>
+    <title>Acknowledgments</title>
+    <link rel="stylesheet" href="../Base.lproj/Acknowledgments.css"/>
+</head>
+
+<body>
+<h1>Acknowledgments</h1>
```

Comment:
Does not need to change to "Acknowledg**e**ments"?

Reply:
Fixed in a2b54e1